src/lib.rs

```diff
@@ -44,17 +44,18 @@ pub mod atomic {
     pub use arc_cell::ArcCell;
 }
 
-/// Scoped thread.
+/// Utilities for concurrent programming.
 ///
 /// See [the `crossbeam-utils` crate](https://github.com/crossbeam-rs/crossbeam-utils) for more
 /// information.
-pub mod thread {
-    pub use crossbeam_utils::thread::*;
+pub mod utils {
+    pub use crossbeam_utils::*;
```

Comment:
Maybe put here only `CachePadded` since `AtomicConsume` is inside `atomic` and `thread` module is already exported?